src/core/test_helpers/kbn_server.ts

```diff
@@ -153,10 +159,7 @@ export function createTestServers({
 }: {
   adjustTimeout: (timeout: number) => void;
   settings?: {
-    es?: {
-      license: 'basic' | 'gold' | 'trial';
-      [key: string]: any;
-    };
+    es?: Partial<CreateTestEsClusterOptions>;
```

Comment:
optional/nice to have: Could we add a bit more to the `settings` prop description about configuring multiple nodes? It feels a bit "hidden" now.

We'll also need to update the testing guide but that could come later.

Reply:
Yep, I can beef up the `CreateTestEsClusterOptions` documentation some more, which should be surfaced here. (For whatever reason couldn't get your link above to load, but I assume that's what you are talking about?)

Reply:
Updated, lmk if you think anything else is missing!